Library/Homebrew/test/test_diagnostic.rb

```diff
@@ -105,11 +105,11 @@ def test_check_user_path_usr_bin_before_homebrew
     sep = File::PATH_SEPARATOR
     # ensure /usr/bin is before HOMEBREW_PREFIX/bin in the PATH
     ENV["PATH"] = "/usr/bin#{sep}#{bin}#{sep}" +
-      ENV["PATH"].gsub(%r{(?:^|#{sep})(?:/usr/bin|#{bin})}, "")
+                  ENV["PATH"].gsub(%r{(?:^|#{sep})(?:/usr/bin|#{bin})}, "")
 
     # ensure there's at least one file with the same name in both /usr/bin/ and
     # HOMEBREW_PREFIX/bin/
-    (bin/"#{File.basename Dir["/usr/bin/*"].first}").mkpath
+    (bin/(File.basename Dir["/usr/bin/*"].first)).mkpath
```

Comment:
Minor nit, but it's nicer to put parentheses around the method argument(s) instead of wrapping the method invocation as a whole in parentheses, i.e.:

```ruby
(bin/File.basename(Dir["/usr/bin/*"].first)).mkpath
```